modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -2858,6 +2872,66 @@ TEST(TestAgeGender, ThrowBlobAndInputPrecisionMismatch)
                      cv::compile_args(cv::gapi::networks(pp))));
 }
 
+#ifdef HAVE_NGRAPH
+
+TEST(Infer, ModelWith2DInputs)
+{
+    const std::string model_name   = "ModelWith2DInputs";
```

Comment:
Is it necessary to add some message about this model or add skip exception? We will use HAVE_NGRAPH=ON and will receive failed test without model. Should i hardcode this path for my own model?

Reply:
This model is created on the fly, I didn't get your question

Reply:
Ok. I saw this.